examples/integrations/eclipsestore/greetings-mp/src/main/java/io/helidon/examples/integrations/eclipsestore/greetings/mp/GreetingProvider.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2021, 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.examples.integrations.eclipsestore.greetings.mp;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+import io.helidon.integrations.eclipsestore.cdi.EclipseStoreStorage;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
```

Comment:
`jakarta` and `org.eclipse` should not be separated.

There are 3 import groups: `java.*`, `io.helidon.*` and everything else

Reply:
Groups are separated by a new line.